OpenCO2_Sensor.ino

```diff
@@ -363,9 +363,10 @@ void initOnce() {
   preferences.end();
 
   scd4x.stopPeriodicMeasurement();  // stop potentially previously started measurement
-  scd4x.getSerialNumber(serial0, serial1, serial2);
```

Comment:
serial0 etc are used in epd_abstraction.ino displayinfo() https://github.com/davidkreidler/OpenCO2_Sensor/blob/main/epd_abstraction.ino#L1167 and is still in the beginning of this file. Proposal add it as RTC_DATA_ATTR  uint64_t serialNumber; and remove https://github.com/davidkreidler/OpenCO2_Sensor/blob/main/OpenCO2_Sensor.ino#L95

Reply:
done in https://github.com/davidkreidler/OpenCO2_Sensor/pull/29/commits/b6d652e562d85d519da5fe7733d4edc34ab73dba